docs/PasswordInput.md

```diff
@@ -47,3 +57,61 @@ Set the [`autocomplete` attribute](https://developer.mozilla.org/en-US/docs/Web/
 <PasswordInput source="password" inputProps={{ autocomplete: 'current-password' }} />
 ```
 {% endraw %}
+
+## Validating Identical Passwords
+
+If you want to validate that the user has entered the same password in two different password inputs, you must use [global validation](./Validation.md#global-validation) at the form level:
```

Comment:
Not really, validators have access to the other form values, so you could write an `equalToFieldValidator`:

```tsx
const equalToFieldValidator = (source: string) => (value: any, allValues: FormValues) => ...
```

Reply:
TIL!